identifiers/country-id/dewan_perwakilan_rakyat.csv

```diff
@@ -64,7 +64,7 @@ ocd-division/country:id/ed:nusa_tenggara_timur_2,Nusa Tenggara Timur 2
 ocd-division/country:id/ed:papua,Papua
 ocd-division/country:id/ed:papua_barat,Papua Barat
 ocd-division/country:id/ed:papua_barat_daya,Papua Barat Daya
-ocd-division/country:id/ed:papua_pengunungan,Papua Pengunungan

```

Comment:
Can you add an alias to mark `ocd-division/country:id/ed:papua_pengunungan` sameAs `ocd-division/country:id/ed:papua_pegunungan`